packages/drivers/odsp-driver/.eslintrc.js

```diff
@@ -13,5 +13,6 @@ module.exports = {
 		"@typescript-eslint/no-use-before-define": "off",
 		"@typescript-eslint/strict-boolean-expressions": "off",
 		"@rushstack/no-new-null": "error",
+		"import/no-deprecated": "warn", // This package uses the deprecated ShareLinkTypes type. Once we remove that, we can remove this override. It's set as a warning instead of "off" to serve as a reminder.
```

Comment:
@jatgarg Do you know if there are plans to update the odsp-driver to use the new types?